source/get-started/primer.txt

```diff
@@ -0,0 +1,157 @@
+   for instructions on how to install the driver.
+
+Reactive Streams
+----------------
+
+This library is an implementation of the reactive
+streams specification. The reactive stream API consists
+of the following components:
+
+1. `Publisher <{+rs-docs+}/Publisher.html>`__
+#. `Subscriber <{+rs-docs+}/Subscriber.html>`__
+#. `Subscription <{+rs-docs+}/Subscription.html>`__
+
+A ``Publisher`` is a provider of a potentially
+unbounded number of sequenced elements, published
+according to the demand received from its ``Subscriber`` or multiple
+instances of ``Subscriber``.
+
+In response to a call to ``Publisher.subscribe(Subscriber)``, the
+possible invocation sequences for methods on the ``Subscriber`` class
+are given by the following protocol:
+
+.. code-block:: java
+   :copyable: false
+
+   onSubscribe onNext* (onError | onComplete)?
+
+This means that ``onSubscribe`` is always
+signaled, followed by a possibly unbounded number
+of ``onNext`` signals, as requested by
+``Subscriber``. This is followed by an ``onError`` signal
+if there is a failure or an ``onComplete`` signal
+when no more elements are available, as long as
+the ``Subscription`` is not canceled.
+
+.. tip::
+
+   To learn more about reactive streams, visit
+   the `Reactive Streams documentation <http://www.reactive-streams.org>`__.
+
+Subscribers
+-----------
+
+The {+driver-short+} API mirrors the :driver:`Java Sync driver
+</java/sync/current/>` API and any methods that cause network 
+I/O return a ``Publisher<T>`` type, where ``T`` is the type of response
+for the operation.
+
+.. note::
+
+   All ``Publisher`` types returned from the API are
+   `cold <https://projectreactor.io/docs/core/release/reference/#reactor.hotCold>`__,
+   meaning that nothing happens until they are subscribed to. So just
+   creating a ``Publisher`` won’t cause any network I/O. It’s not until
+   you call the ``Publisher.subscribe()`` method that the driver executes
+   the operation.
+   
+   Publishers in this implementation are *unicast*. Each
+   ``Subscription`` to a ``Publisher`` relates to a single MongoDB
+   operation, and the ``Publisher`` instance's ``Subscriber`` receives its
+   own specific set of results.
+
+Subscribers in the Quick Start
+------------------------------
+
+In the Quick Start, we have implemented a couple of
+different ``Subscriber`` types. Although this is an
+artificial scenario for reactive streams we do
+block on the results of one example before starting
+the next, so as to ensure the state of the
+database.
+
+1. ``ObservableSubscriber``: The base subscriber class is the
+   `ObservableSubscriber<T> <{+driver-source-gh+}/blob/master/driver-reactive-streams/src/examples/reactivestreams/helpers/SubscriberHelpers.java>`__,
+   a ``Subscriber`` that stores the results of the ``Publisher<T>``. It
+   also contains an ``await()`` method so we can block for results
+   to ensure the state of the database before going
+   on to the next example.
```

Comment:
Suggestion: use an unordered list and use the "definition" spacing (everthing below the first line is indented an extra space)
```suggestion
- ``ObservableSubscriber``
    The base subscriber class is the
   `ObservableSubscriber<T> <{+driver-source-gh+}/blob/master/driver-reactive-streams/src/examples/reactivestreams/helpers/SubscriberHelpers.java>`__,
   a ``Subscriber`` that stores the results of the ``Publisher<T>``. It
   also contains an ``await()`` method so we can block for results
   to ensure the state of the database before going
   on to the next example.
```